applications/SwimmingDEMApplication/custom_elements/calculate_velocity_laplacian.cpp

```diff
@@ -52,20 +52,14 @@ int ComputeVelocityLaplacianSimplex<TDim, TNumNodes>::Check(const ProcessInfo& r
     int ErrorCode = Kratos::Element::Check(rCurrentProcessInfo);
     if(ErrorCode != 0) return ErrorCode;
 
-    if(this->GetGeometry().size() != TDim+1)
-        KRATOS_THROW_ERROR(std::invalid_argument,"wrong number of nodes for element",this->Id());
-
-    if (MATERIAL_ACCELERATION.Key() == 0){
```

Comment:
@GuillermoCasas this was bug, no?

Reply:
probably since MATERIAL_ACCELERATION  is being checked in the previous file